style.css

```diff
@@ -131,17 +166,36 @@ img {
 }
 
 .nav-list {
-  display: none;
+  display: flex;
+  position: fixed;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  text-align: center;
+  left: -100%;
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  color: #fff;
+  top: 0;
+  gap: 10px;
+  padding-top: 70px;
 }
 
-.nav-item {
-  list-style: none;
-  margin-right: 2rem;
+.nav-list.active {
+  left: 0;
 }
 
-.nav-item a {
+.nav-item {
   text-decoration: none;
-  color: #344563;
+  color: #fff;
+  list-style: none;
+  margin-right: 2rem;
+  font-family: "Poppins", sans-serif !important;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 50px;
+  text-align: left;
+  padding-left: 20px;
 }
```

Comment:
- [ ] Please, consider this requirement style from the Figma template:

> Figma design
<img width="203" alt="Screen Shot 2022-09-06 at 13 29 01" src="https://user-images.githubusercontent.com/49064106/188601987-2c555fbe-3008-48d2-8d43-59034badfbe1.png">

You can see that some of the properties (such as `font-weight`, `line-height`, etc) are different from your one